app/models/like.rb

```diff
@@ -0,0 +1,3 @@
+class Like < ApplicationRecord
+  belongs_to :user, :post
+end
```

Comment:
As part of the project requirement, Let's add a method that updates the likes counter for a post in the `like` model

> ![image](https://user-images.githubusercontent.com/66112760/150019811-5919db63-cfa8-4939-8e71-7d40018a4dfd.png)